neo/Prompt/Commands/Send.py

```diff
@@ -35,60 +96,58 @@ def construct_send_basic(wallet, arguments):
     assetId = get_asset_id(wallet, to_send)
     if assetId is None:
         print("Asset id not found")
-        return False
+        return
 
     scripthash_to = lookup_addr_str(wallet, address_to)
     if scripthash_to is None:
         logger.debug("invalid address")
-        return False
+        return
 
     scripthash_from = None
     if from_address is not None:
         scripthash_from = lookup_addr_str(wallet, from_address)
         if scripthash_from is None:
             logger.debug("invalid address")
-            return False
+            return
 
     # if this is a token, we will use a different
     # transfer mechanism
     if type(assetId) is NEP5Token:
-        return do_token_transfer(assetId, wallet, from_address, address_to, amount_from_string(assetId, amount), tx_attributes=user_tx_attributes)
+        return do_token_transfer(assetId, wallet, from_address, address_to, amount_from_string(assetId, amount),
+                                 tx_attributes=user_tx_attributes)
 
     f8amount = get_asset_amount(amount, assetId)
     if f8amount is False:
         logger.debug("invalid amount")
-        return False
+        return
     if float(amount) == 0:
         print("amount cannot be 0")
-        return False
+        return
 
     fee = Fixed8.Zero()
     if priority_fee is not None:
         fee = priority_fee
         if fee is False:
             logger.debug("invalid fee")
-            return False
+            return
 
     output = TransactionOutput(AssetId=assetId, Value=f8amount, script_hash=scripthash_to)
     contract_tx = ContractTransaction(outputs=[output])
     return [contract_tx, scripthash_from, fee, owners, user_tx_attributes]
 
 
 def construct_send_many(wallet, arguments):
-    if not wallet:
-        print("please open a wallet")
-        return False
     if len(arguments) is 0:
         print("Not enough arguments")
-        return False
+        return
```

Comment:
Should return something here too.

Reply:
https://github.com/CityOfZion/neo-python/pull/726#discussion_r236978513